app/test_main.py

```
@@ -1,3 +1,19 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ]
+)
+def test_get_human_age(cat_age : int, dog_age : int, expected : list) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

Good work on testing the happy-path scenarios and boundary conditions like ages 14, 15, 24, etc. 

However, you've missed some important edge cases required by the task checklist. This violates checklist item #2: "Make sure you've tests checks all edged situations such as: ... if function resive data out of normal range, such as negative numbers ... if the function receives an incorrect type of data it raises the correct exception."

Your test suite should also include cases for:
1.  **Negative numbers**: What should happen if `cat_age` or `dog_age` is negative?
2.  **Incorrect data types**: What should happen if strings or floats are passed instead of integers? The function should raise an appropriate exception like `TypeError`.

You can create another test function to handle these error cases. For testing exceptions, you can use `pytest.raises`. For example:

```python
import pytest

def test_get_human_age_invalid_input():
    with pytest.raises(TypeError):
        get_human_age("cat", "dog")
```